config/routes.rb

```diff
@@ -35,4 +35,6 @@
   if Rails.env.development?
     mount GovukAdminTemplate::Engine, at: "/style-guide"
   end
+
+  mount Proxies::GovernmentProxy.new => Proxies::GovernmentProxy::PROXY_BASE_PATH
```

Comment:
Can we lock this down to authenticated users? You can create a constraint that verifies `request.env['warden'].user` isn't nil. See collections-publisher for an example:

https://github.com/alphagov/collections-publisher/blob/master/config/routes.rb#L39-L49